examples/sol-contract/src/processor.rs

```diff
@@ -0,0 +1,67 @@
+//! Program instruction processor
+
+use solana_program::msg;
+use solana_program::pubkey::Pubkey;
+use solana_program::account_info::AccountInfo;
+use solana_program::entrypoint::ProgramResult;
+use solana_program::program_error::ProgramError;
+
+use borsh::BorshDeserialize;
+use crate::instruction::PythClientInstruction;
+use pyth_sdk_solana::load_price_feed_from_account_info;
+
+pub fn process_instruction(
+    _program_id: &Pubkey,
+    _accounts: &[AccountInfo],
+    input: &[u8],
+) -> ProgramResult {
+    // Checking the validity of parameters is important.
+    // This important step is skipped in this example contract.
+    let instruction = PythClientInstruction::try_from_slice(input)?;
+    match instruction {
+        PythClientInstruction::Loan2Value {} => {
+            let loan = &_accounts[0];
+            let collateral = &_accounts[1];
+            msg!("The loan key is {}.", loan.key);
+            msg!("The collateral key is {}.", collateral.key);
+            msg!("Assume 1 unit of loan and 3000 unit of collateral.");
+            let loan_cnt = 1;
+            let collateral_cnt = 3000;
```

Comment:
nitpick: would suggest `loan_qty` and `collateral_qty`. `cnt` is a bit confusing